node/src/main/scala/com/wavesplatform/api/http/TransactionsApiRoute.scala

```diff
@@ -66,26 +66,30 @@ case class TransactionsApiRoute(
     } ~ path(TransactionId) { id =>
       commonApi.transactionById(id) match {
         case Some((h, either, succeeded)) =>
-          complete(txToExtendedJson(either.fold(identity, _._1)) ++ applicationStatusJsField(isBlockV5(h), succeeded) + ("height" -> JsNumber(h)))
+          val tx = either.fold(identity, _._1)
+          complete(txToExtendedJson(tx) ++ enrich(tx) ++ applicationStatusJsField(isBlockV5(h), succeeded) + ("height" -> JsNumber(h)))
         case None => complete(ApiError.TransactionDoesNotExist)
       }
     }
   }
 
+  private def enrich(t: Transaction) = {
+    t match {
+      case t: smart.InvokeScriptTransaction => Json.obj("сontinuationTransactionIds" -> commonApi.continuations(t.id()).map(_.toString))
```

Comment:
It is unnecessary field for old InvokeScript transactions that don't support continuations. Introduce InvokeScript transaction V3 to distinguish.